### ISSUE TITLE
[Feature Request] export checklist content as plain text

### ISSUE THREAD

**Issue by RealJTG** [SimpleMobileTools/Simple-Notes#533]
Please consider adding ability to export checklist as plaintext. Currently I can export checklist only as JSON:
```json
[{"dateCreated":1653907613810,"id":1,"isDone":false,"title":"Foo"},{"dateCreated":1653907613810,"id":2,"isDone":true,"title":"Bar"},{"dateCreated":1653907613810,"id":3,"isDone":false,"title":"Baz"}]
```

I'd like to get human-readable text for sharing with someone on messengers etc., could be something like this:

```
☐ Foo
☑ Bar
☐ Baz
```
or
```
✘ Foo
✔ Bar
✘ Baz
```

**Comment by bigConifer** [comment]
Yes, it would be nice if at least the share option sends plain text. However, I think there shouldn't be an additional option in the menu.

Instead of using special characters, [Markdown task list](https://www.markdownguide.org/extended-syntax/#task-lists) is a pretty suitable format for this:
`- [ ] Foo`
`- [x] Bar` 
It's easy to read and to edit. And when you open the exported text file in a Markdown application, the items get rendered as checkboxes. Even [GitHub Flavored Markdown](https://github.github.com/gfm/#task-list-items-extension-) supports it:
- [ ] Foo
- [x] Bar

**Comment by Filler-dev** [comment]
@tibbi Can you pls add this feature
